src/vcInternalTexturesData.h

```diff
@@ -0,0 +1,6 @@
0x49, 0x3c, 0xf6, 0x4b, 0xc9, 0x72, 0x41, 0xa8, 0x77, 0xd1, 0xfa, 0xd1, 0x3d, 0x37, 0x31, 0x3a, 0xfa, 0x56, 0x2c, 0x0b, 0x6c, 0x3b, 0x48, 0xc0, 0x6e, 0xc6, 0xb6, 0x17, 0x63, 0x59, 0xed, 0xd4, 0x8d, 0x26, 0xb9, 0xd2, 0x0d, 0xfc, 0x2d, 0x6a, 0xed, 0xc0, 0x51, 0xc9, 0x6e, 0x41, 0x68, 0x04, 0xd1, 0x9a, 0xd9, 0x6a, 0x3c, 0x7e, 0xa5, 0x04, 0xdc, 0x01, 0x7c, 0xe9, 0x65, 0x0b, 0x64, 0x0a, 0x82, 0x50, 0xd7, 0xd8, 0x0d, 0x18, 0xa7, 0x0d, 0xa8, 0x7e, 0xab, 0x47, 0x24, 0x7b, 0x05, 0x41, 0x44, 0xcb, 0xcf, 0x3c, 0x0b, 0x5c, 0x47, 0x3c, 0xf6, 0xaf, 0x92, 0xad, 0x82, 0x20, 0xa2, 0xe5, 0x67, 0x06, 0x80, 0xaf, 0x03, 0xb7, 0x12, 0x8f, 0x0d, 0x4a, 0x96, 0x0a, 0x82, 0x88, 0x96, 0x9f, 0xf9, 0x09, 0xf0, 0x39, 0xe2, 0xb1, 0xe7, 0x24, 0x2b, 0x05, 0x41, 0x44, 0xcb, 0xcf, 0x3c, 0x8c, 0x9a, 0x7a, 0xf3, 0x5b, 0xc9, 0x42, 0x41, 0x98, 0x5f, 0x98, 0x75, 0x76, 0xbd, 0xfb, 0x80, 0x8f, 0x02, 0x6b, 0x45, 0xb0, 0x04, 0x41, 0x6a, 0x5a, 0x7e, 0x66, 0x18, 0xf8, 0x26, 0x70, 0x83, 0x58, 0xc6, 0x08, 0x82, 0x88, 0x96, 0xdf, 0xe9, 0x02, 0x3e, 0x2d, 0x96, 0x31, 0x82, 0x20, 0xf8, 0x5d, 0xb4, 0xfe, 0x80, 0xea, 0xb7, 0x7a, 0x40, 0xb2, 0x49, 0x10, 0x84, 0x31, 0xfc, 0xd8, 0xa7, 0x75, 0x18, 0xf8, 0x38, 0xb0, 0x5a, 0x04, 0x4b, 0x10, 0x04, 0x3f, 0xd7, 0xb4, 0x0a, 0xc0, 0x77, 0x50, 0x96, 0x31, 0x47, 0x25, 0x6b, 0x04, 0x41, 0xf0, 0xb3, 0x68, 0xdd, 0x0d, 0x7c, 0x8a, 0x78, 0xec, 0x8f, 0x92, 0x25, 0x82, 0x20, 0xf8, 0x59, 0xb4, 0x9e, 0xd4, 0x62, 0xf5, 0x0b, 0xc9, 0x0a, 0x41, 0x10, 0x2a, 0x61, 0xae, 0xfa, 0xb4, 0x7a, 0x80, 0x4f, 0x03, 0x67, 0x8b, 0x60, 0x09, 0x82, 0xe0, 0xe7, 0x9a, 0x96, 0x83, 0xb2, 0x8c, 0xf9, 0xa2, 0x58, 0xc6, 0x08, 0x82, 0xe0, 0x77, 0xd1, 0xfa, 0x0d, 0xf0, 0x09, 0xb1, 0x8c, 0x11, 0x04, 0xc1, 0xef, 0xa2, 0xb5, 0x13, 0x65, 0x19, 0xf3, 0x2f, 0x92, 0xdc, 0x82, 0x20, 0xf8, 0x59, 0xb4, 0x06, 0x80, 0x9b, 0x80, 0xbf, 0x17, 0xcb, 0x18, 0x41, 0x10, 0xfc, 0x2e, 0x5a, 0x3f, 0xd5, 0xb5, 0x2b, 0xb1, 0x8c, 0x11, 0x04, 0xc1, 0xd7, 0xa2, 0xf5, 0xdf, 0xa8, 0xa9, 0x37, 0x79, 0x49, 0x5a, 0x41, 0x10, 0x66, 0x03, 0xaf, 0x86, 0x3c, 0xec, 0x03, 0xae, 0x04, 0xce, 0x17, 0xc1, 0x12, 0x04, 0xc1, 0xcf, 0x35, 0xad, 0x31, 0xcb, 0x98, 0xaf, 0x13, 0x8f, 0xf5, 0x4a, 0x72, 0x0a, 0x82, 0xe0, 0x67, 0xd1, 0xfa, 0x19, 0xca, 0x32, 0xe6, 0x19, 0x49, 0x46, 0x41, 0x10, 0xfc, 0x2c, 0x5a, 0x8f, 0xa2, 0xfa, 0xad, 0xee, 0x97, 0xe4, 0x13, 0x04, 0xa1, 0xd6, 0x54, 0xd3, 0xa7, 0xf5, 0x3c, 0x2f, 0x5a, 0xc6, 0x88, 0x60, 0x09, 0x82, 0xe0, 0xdb, 0x9a, 0x56, 0x11, 0x65, 0x19, 0xf3, 0x55, 0xb1, 0x8c, 0x11, 0x04, 0xc1, 0xef, 0xa2, 0x75, 0x2f, 0x6a, 0xb5, 0x66, 0xb1, 0x8c, 0x11, 0x04, 0xc1, 0xd7, 0xa2, 0xb5, 0x1d, 0x65, 0x19, 0xf3, 0x1f, 0x92, 0x44, 0x82, 0x20, 0xf8, 0x89, 0x89, 0x7d, 0x5a, 0x3d, 0xc0, 0x67, 0x80, 0x95, 0x22, 0x58, 0x82, 0x20, 0xf8, 0xb9, 0xa6, 0xe5, 0x00, 0xff, 0x08, 0x7c, 0x41, 0x2c, 0x63, 0x04, 0x41, 0xf0, 0xbb, 0x68, 0xfd, 0x06, 0x35, 0x84, 0x61, 0xb3, 0x24, 0x87, 0x20, 0x08, 0x7e, 0xe7, 0xff, 0x0f, 0x00, 0x2d, 0x3b, 0x36, 0xec, 0x36, 0x31, 0x0e, 0x0c, 0x00, 0x00, 0x00, 0x00, 0x49, 0x45, 0x4e, 0x44, 0xae, 0x42, 0x60, 0x82 }
```

Comment:
Can I veto this awful practice of jamming multiple file blobs into a single header?

Reply:
You'd prefer single file blobs in a header? Paul specified on AB to embed the file so it's not easy to change

Reply:
> You'd prefer single file blobs in a header?

No, data doesn't need to be in headers and thus I'd prefer it if the data wasn't in the header.

> Paul specified on AB to embed the file so it's not easy to change

I know, but that doesn't require the data to be in the header.

Reply:
The other blobs are in `.cpp` files with externs in the header

Reply:
Not the `vcInternalModelsData.h`, the `vcInternalModels.h`/`.cpp` do but I think they're loaded not compiled in.